.github/workflows/cicd.yml

```diff
@@ -49,7 +48,9 @@ jobs:
           python -m pip install -U pip --no-cache-dir
           python -m pip install -e . -r dev_requirements.txt
       - name: Test
-        run: invoke test
+        env:
+          CODECOV_TOKEN: ${{ secrets.CODECOV_TOKEN }}
+        run: invoke test --coverage
```

Comment:
Is this to capture the Windows case which might go down a different branch? Otherwise seems redundant to collect coverage multiple times.

Reply:
It's to cover both paths  where`mypy_extension` is and is not available e.g. in the minimum tests